workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/StandardGameObjectCreation/GameObjectCreationSystemHelper.cs

```diff
@@ -0,0 +1,18 @@
+using Unity.Entities;
+
+namespace Improbable.Gdk.Core.GameObjectRepresentation
+{
+    public static class GameObjectCreationSystemHelper
+    {
+        public static void AddStandardGameObjectCreation(World world)
+        {
+            string workerType = world.GetOrCreateManager<WorkerSystem>().WorkerType;
+            AddStandardGameObjectCreation(world, new GameObjectCreatorFromMetadata(workerType));
+        }
+
+        public static void AddStandardGameObjectCreation(World world, IEntityGameObjectCreator creator)
```

Comment:
Maybe rename this to something like `SetEntityGameObjectCreator()`? "Add" kinda implies that you could potentially have multiple creators?

Reply:
It's meant to convey that you're adding this sub feature module thing of us creating the gameobjects for you. Maybe EnableStandardGameObjectCreation, would that sound good? I'd like the two methods to be the same, the other doesn't have the parameter but defaults to our implementation